### DISCUSSION TITLE
Recurrente task &email notification

### DISCUSSION THREAD

**Discussion by nykroy** [MDeLuise/plant-it#98] [Feature Request]
### Avoid duplicated feature requests

- [X] There are no open or closed feature requests that are related to this request

### Description

It would be a good options:
1. Smtp configuration for email notifications
2. Recurrent tasks

**Comment by MDeLuise** [comment]
Hi @nykroy,
thanks for opening a feature request.

Implementing SMTP server/configuration is a great idea. It would enhance the platform's functionality, especially for processes like user signup and password changes.

Regarding recurrent tasks, what does you mean exactly? Something like "auto log that I water this plant every 2 days"?

**Reply by nykroy** [reply]
Exactly you can add the recurrent tasks for wáter some plant every 7 days and when the target day IS the selected by email the system write you a notifications for reminder the task to do.
I think it is a good idea for the system to notify you until you mark the task as done and not use the app only as a log of actions but also as tasks done or to be done.  So, for example, for bonsais you can plan pinching, fertilizing or transplanting.

**Reply by MDeLuise** [reply]
Absolutely, this is indeed a great idea.
From the beginning of the project, implementing a notification feature has been on my radar, allowing users to set reminders for various plant-related tasks.
Initially, I thought of having notifications sent directly through the app. However, provide the option to receive notifications via both the app and email (based on the user preference) is great. This would enhance the user experience and ensure that reminders are easily accessible.

**Comment by nykroy** [comment]
You can see how this another similar app had implemented the email config. https://github.com/danielbrendel/hortusfox-web. I thing both goes on the same line of type off app :)

**Reply by MDeLuise** [reply]
I will surely take a look thanks!
